### ISSUE TITLE
Measure Repeats do not import, etc.

### ISSUE THREAD

**Issue by mscuthbert** [cuthbertLab/music21#351]
A missing feature that should not have persisted this long.

**Comment by cahille** [comment]
Does this issue explain why I see these measures when I view my musicxml file

<img width="615" alt="image" src="https://user-images.githubusercontent.com/10825734/174661803-13668ba6-6c4c-4344-aff2-17a634b6ae1d.png">

but when I just import the above file and show (converter.parse(path, forceSource=True).show()) the repeats are gone?

<img width="562" alt="image" src="https://user-images.githubusercontent.com/10825734/174662043-fb804fb1-5df9-4616-ad7e-29f33be86a4f.png">

Thanks!
Earl

**Comment by mscuthbert** [comment]
No, that is a different issue.  These "endings" should import properly if they are encoded properly in MusicXML.  Look to see if your MusicXML export imports properly in another reader.  This issue refers to the "Repeat last measure" sign.
